specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-03-01/orbital.json

```diff
@@ -1360,15 +1360,15 @@
             "providerCancelled"
           ],
           "x-ms-enum": {
-            "name": "Status",
-            "modelAsString": false
+            "name": "ContactsStatus",
+            "modelAsString": true
```

Comment:
This is the only change that affect SDK.

Status -> ContactsStatus is agreed by owner of released SDKs, as it fixes the duplication from another "Status".

I am not aware of the change in "modelAsString". This should be the correct value, but we need to get approval from Go. @tadelesh

Reply:
You should be able to delete the "directive" in readme.go.md
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/orbital/resource-manager/readme.go.md

Reply:
Thanks @weidongxu-microsoft. Changing to `modelAsString: true` won't cause breaking in Go.